RallyHereIntegration/Source/RallyHereIntegration/Private/RH_Integration.cpp

```diff
@@ -75,8 +75,12 @@ static FAutoConsoleCommandWithWorldArgsAndOutputDevice ConsoleRHSetEnvironmentId
 			NewValue.TrimQuotesInline();
 			FRallyHereIntegrationModule::Get().LockEnvironmentId(bHasValue);
 			FRallyHereIntegrationModule::Get().SetEnvironmentId(MoveTemp(NewValue), TEXT("Console Command"));
-			FRallyHereIntegrationModule::Get().LockBaseURL(false);
```

Comment:
While I get that this works without it - if a user is changing the environment, I can't imagine that they want anything less than all of the associated fields for the environment.  By not forcibly unlocking the base URL, it's possible to lock the base URL and change the environment without having the URL change - which feels really wrong.